docs/source/api/graphstorm.rst

```diff
@@ -9,13 +9,14 @@ graphstorm
     Users can directly use the following code to use these functions.
 
     >>> import graphstorm as gs
-    >>> gs.initialize()
-    >>> gs.get_rank()
+    >>> gs.initialize(ip_config="/tmp/ip_list.txt", backend="gloo")
+    >>> gs.setup_device(local_rank)
+    >>> gs.get_feat_size(g, "ntyp1")
```

Comment:
Why users need to call gs.get_feat_size(g, "ntyp1") ?

Only the first two are recommended.

Reply:
Some old codes call it. No need anymore. Removed.